packages/esm-home-app/src/routes.json

```diff
@@ -32,6 +32,13 @@
       "online": true,
       "offline": true
     },
+    {
+      "name": "home-page-header",
+      "slot": "home-dashboard-slot",
```

Comment:
This means it will be displayed by default

Reply:
Yes, the essence of the header is to be static but configurable.